plugin-api.d.ts

```diff
@@ -1737,8 +1755,6 @@ interface Variable {
   readonly id: string
   name: string
   description: string
-  hiddenFromPublishing: boolean
-  getPublishStatusAsync(): Promise<PublishStatus>
```

Comment:
Can you pull in the latest changes from master so we don't have these deletions?

Reply:
oops, thanks for catching this!